components/NotificationListItem.tsx

```
@@ -0,0 +1,165 @@
+          <>
+            <TitleText>{item.content.body ?? '새로운 피드'}</TitleText>
+            <SubText>{item.content.authorName ?? '내용 없음'}</SubText>
+          </>
+        );
+
+      case 'FEED_TAG':
+        const name = item.content.authorName ?? '내용 없음';
+        const josa = getSubjectJosa(name);
+        return (
+          <>
+            <TitleText>{`${name}${josa} 나를 게시글에 언급했어`}</TitleText>
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)